hazelcast/src/main/java/com/hazelcast/internal/partition/impl/InternalPartitionServiceImpl.java

```diff
@@ -666,6 +659,11 @@ private void finalizeOrRollbackMigration(PartitionRuntimeState partitionState) {
         }
 
         updateAllPartitions(partitionTable);
+        partitionStateManager.setVersion(partitionState.getVersion());
+        if (!partitionStateManager.setInitialized()) {
```

Comment:
`partitionStateManager.setInitialized()` already calls `node.getNodeExtension().onPartitionStateChange()` if it sets `initialized=true`. This is just to remove 2nd redundant `node.getNodeExtension().onPartitionStateChange()` call. But I can remove `if` check completely. It doesn't make so much diff performance-wise.